### PR TITLE
[IMP] hr_expense : fix issues

### PR DESCRIPTION
**Pull request by anr-odoo**
Task : https://www.odoo.com/web#id=2061910&action=327&model=project.task&view_type=form&menu_id=4720

Pad: https://pad.odoo.com/p/r.d6e0e949ee8448526bc9e87791bda451

Description of the issue/feature this PR addresses:

Current behavior before PR:

Desired behavior after PR is merged:

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/hr_expense/views/hr_expense_views.xml [hunk 1]
```diff
@@ -565,13 +567,15 @@
...
                     <field name="state"/>
+                    <field name="currency_id"/>
                     <templates>
                         <t t-name="kanban-box">
                             <div t-attf-class="oe_kanban_card oe_kanban_global_click">
                                 <div class="row">
                                     <div class="col-12">
                                         <strong class="o_kanban_record_title"><span><t t-esc="record.name.value"/></span></strong>
                                         <strong class="o_kanban_record_subtitle float-right"><span class="text-right"><t t-esc="record.total_amount.value"/></span></strong>
+                                        <strong class="o_kanban_record_subtitle float-right mr4"><span class="text-right"><t t-esc="record.currency_id.value"/></span></strong>
```
Comment (jem-odoo) [inline addons/hr_expense/views/hr_expense_views.xml]: can you use the 'monetary' widget on the amount field ? in order to have the currency symbol on the kanban card ? Thanks

File: addons/hr_expense/views/hr_expense_views.xml [hunk 2]
```diff
@@ -154,6 +155,7 @@
...
```
Comment (jem-odoo) [inline addons/hr_expense/views/hr_expense_views.xml]: can you use the 'monetary' widget on the amount field ? in order to have the currency symbol on the kanban card ? Thanks

File: addons/hr_expense/views/hr_expense_views.xml (hunk 1 above)
Comment (anr-odoo) [reply addons/hr_expense/views/hr_expense_views.xml]: @jem-odoo done.

**Comment by jem-odoo** [comment]
@robodoo r+

**Pull request closed without merging**